jsk_2016_01_baxter_apc/robots/baxter.yaml

```diff
@@ -58,6 +58,7 @@ angle-vector:
   view-hand-pose-j: [0, 65.2368, 19.5557, -78.2886, 121.069, 44.4946, 18.3911, 74.0698, 90, -46.3843, 29.0918, 71.4551, 121.794, -48.4058, 13.8647, -79.0356, 90]
   view-hand-pose-k: [0, 46.3843, 29.0918, -71.4551, 121.794, 48.4058, 13.8647, 79.0356, 90, -46.3843, 29.0918, 71.4551, 121.794, -48.4058, 13.8647, -79.0356, 90]
   view-hand-pose-l: [0, 46.3843, 29.0918, -71.4551, 121.794, 48.4058, 13.8647, 79.0356, 90, -65.2368, 19.5557, 78.2886, 121.069, -44.4946, 18.3911, -74.0698, 90]
+  approach-from-downside: [0, 34.8926, 7.64648, -7.82227, 42.2095, -56.7773, 102.986, -23.9722, 90, -34.8926, 7.64648, 7.82227, 42.2095, 56.7773, 102.986, 23.9722, 90]
```

Comment:
from upper to lower pose, baxter needs this midpoint otherwise it conflict with tote
![view_hand_pose](https://cloud.githubusercontent.com/assets/9300063/19549267/53e645aa-96de-11e6-94b2-bc7efe98f808.png)
![hold_opposite_hand_object](https://cloud.githubusercontent.com/assets/9300063/19558548/23bead36-9706-11e6-8249-31349656ac7e.png)